addons/account/wizard/account_move_send.py

```diff
@@ -300,6 +293,28 @@ def _compute_mail_attachments_widget(self):
     # BUSINESS ACTIONS
     # -------------------------------------------------------------------------
 
+    @api.model
+    def action_open_partners_without_email(self, res_ids=None):
+        partners = self.env[self._name].browse(res_ids).move_ids.mapped("partner_id")
```

Comment:
1. Why not just make the action not `@api.model` and use `self.move_ids` here ?
This browse would therefore not be necessary.
2. This is retrieving all partners from all moves, we should only display partners without email :
`self.move_ids.filtered(lambda move: not move.partner_id.email)`